content/11.raw-data-analysis.md

```diff
@@ -1,9 +1,54 @@
 ## Analysis of Raw Data {.page_break_before}
 
-The goal of basic data analysis is to convert raw spectral data into identities and quantities of peptides and proteins that can be used for biologically-focused analysis.
+The goal of basic data analysis is to convert raw spectral data into identities and quantities of peptides and proteins that can be used for biologically-focused analysis. 
+This step may often include measures of quality control, cross-run data normalization, quantification on different levels (precursor, peptide, protein), protein inference, PTM (post translational modification) localization and also first steps of data analysis, such as statistical hypothesis tests. 
+
+In typical bottom-up proteomics experiments, proteins are digested into peptides and further analyzed with LC-MS/MS systems. 
+Peptides can have different PTMs and ionize differently depending on their length and amino acid distributions. 
+Therefore, mass spectrometers often record different charge and modification states of one single peptide. 
+The entity that is recorded on a mass spectrometer is usually referred to as a precursor ion (peptide with its modification and charge state). 
+This precursor ion is fragmented and the precursor or peptide sequences are obtained though spectral matching. 
+The quantity of a precursor is estimated with various methods.
+The measured precursor quantities are combined to generate a peptide quantity. 
+Peptides are also often combined into a protein group through protein inference, which combines multiple peptide identifications into a single protein identification [@DOI:https://doi.org/10.1074/mcp.R500012-MCP200] [@DOI:10.1016/j.jprot.2016.08.002].
+Protein inference is still a challenge in bottom-up proteomics. 
+
+Due to the inherent differences in the data structures of DDA and DIA measurements, there exist different types of software that can facilitate the steps mentioned above. 
+The existing software for DDA and DIA analysis can be further divided into freeware and non-freeware:
+
+<!-- TODO: should we make a table for this part? with primary citation and link to the website -->
+
+DDA freeware: 
+- MaxQuant [@URL:https://www.maxquant.org/]
+- MSFragger [@URL:https://msfragger.nesvilab.org/] 
+- Mascot (for smaller data sets) [@URL:https://www.matrixscience.com/] [@PMID:10612281]
+- MS-GF+ [@DOI:https://doi.org/10.1038/ncomms6277]
```

Comment:
DOI citations expect just the DOI, not the doi.org DOI URL.  So like `@doi:10.1038/ncomms6277`.

The way this is cited still works because of redundancy built in, but it will be slower to build the manuscript and may be more error prone when retrieving the reference metadata.